### DISCUSSION TITLE
Идеи по улучшению радара

### DISCUSSION THREAD

**Discussion by osipxd** [RedMadRobot/techradar-android#10] [Ideas]
Сюда можно писать любые идеи как сделать наш радар лучше.

**Comment by sonulen** [comment]
на страницу о радаре добавить все основные ссылки (на CONTRIBUTING guide, на discussions, на issue и мб форму для них сделать бы)

**Reply by osipxd** [reply]
Cо ссылкой на создание issue бы повременил так как не понятно что в них должно быть, а с остальным согласен.

**Comment by osipxd** [comment]
Переименовать квадрант Utilities в Tools. Помню, что мы для чего-то переименовывали наоборот, но теперь уже не могу вспомнить для чего
